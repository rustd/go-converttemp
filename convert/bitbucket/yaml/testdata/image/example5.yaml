image:
  name: my-account/bash:latest

pipelines:
  default:
    - step:
        script:
          - echo "Hello, World!"
