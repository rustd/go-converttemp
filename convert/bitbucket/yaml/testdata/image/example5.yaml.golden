{
    "image": "my-account/bash:latest",
    "pipelines": {
        "default": [
            {
                "step": {
                    "script": [
                        "echo \"Hello, World!\""
                    ]
                }
            }
        ]
    }
}